homeassistant/util/color.py

```diff
@@ -600,3 +600,25 @@ def check_point_in_lamps_reach(p: Tuple[float, float],
     t = cross_product(v1, q) / cross_product(v1, v2)
 
     return (s >= 0.0) and (t >= 0.0) and (s + t <= 1.0)
+
+
+def check_valid_gamut(Gamut: GamutType) -> bool:
+    """Check if the supplied gamut is valid."""
+    # Check if gamut is None
+    if not Gamut:
```

Comment:
We shouldn't check this in here, instead we should not call this function if it's `None`